docs/UserGuide.md

```diff
@@ -108,15 +114,20 @@ Adds a resident and their information to the application.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL r/ROOM v/VACCINATION_STATUS f/FACULTY [fd/LAST_FET_DATE] [cd/LAST_COLLECTION_DATE]`
 
+Parameter | Constraints
+|--------|-------
+**NAME** | Unique, only containing alphabetical characters and spaces
+**PHONE** | At least 3 digits long
+**ROOM** | consists of **block** + **level** + **number** <br>**block** is an alphabetical character from A to E <br>**level** is a digit from 1 to 4 <br>**number** is two digits from 00 to 29 <br>e.g. `A100`
+**VACCINATION_STATUS** | `T` or `F` (case insensitive)
+**FACULTY** | Single alphabetical word
+**LAST_FET_DATE** <br> **LAST_COLLECTION_DATE** | `dd-mm-yyyy`, `dd.mm.yyyy` or `dd/mm/yyyy` format
+
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Parameter constraints:**<br>
-* `NAME` has to be unique.
-* `PHONE` is at least 3 digits long.
-* `ROOM` is of the form `BLOCK` + `LEVEL` + `NUMBER`, where `BLOCK` is an alphabet, `LEVEL` and `NUMBER` make up 3 digits. e.g. `A100`
-* `VACCINATION_STATUS` can be `T` or `F` (case insensitive).
-* `FACULTY` has to be a single alphabetical word.
-* The dates inputted for `LAST_FET_DATE` and `LAST_COLLECTION_DATE` has to be in dd-mm-yyyy format.
+**:information_source: Note:**<br>
+
+**ROOM** is currently modelled after an existing hall in the National University of Singapore, options for more flexibility will be made in the future.
```

Comment:
I dont think we should explicitly say flexibility will be provided in the future. What do you think?

Reply:
I've removed the part on "options for more flexibility will be made in the future."